### DISCUSSION TITLE
Stop other users from editing my soul link tracker

### DISCUSSION THREAD

**Discussion by ChrissLow** [jynnie/soullocke#22] [Ideas]
So basically I would add a command to my twitch bot to share the link for my soul link tracker, so that any viewer can check the pairs, where we've encountered, what's in our boxes. But I want to make sure no one interferes with our tracker and trolls us. I would love a way for me to lock other users from editing it unless I give them permission.

**Comment by jynnie** [comment]
Would an acceptable way to accomplish this be having a password that is required to edit your tracker? Or would you want accounts and specific share permissions with accounts?

**Reply by ChrissLow** [reply]
Definitely the Permissions option. I'd want my viewers to be able to look at what pairs we have obtained without having immediate access to editing it, like a read only setting. I'd also like giving my soul partner some form of perms to be able to edit it themselves.

**Reply by jynnie** [reply]
Gotcha! That'll be tougher to implement than the password option and my main job has been busy lately, but it has been on my radar for some time. I'll update this thread when I find some time to begin working on that.

**Reply by ChrissLow** [reply]
No worries! Thank you for the update. If the perms option doesn't work out than the password option would be good to have anyways. Whatever is easier for you to implement in the long run.
